### PR TITLE
Teleporter code cleanup / test fire tweak

### PR DESCRIPTION
**Pull request by Intigracy**
~~Removes a bunch of unused code (like the var that only existed for the clown planet card which got removed)~~

Removes an unused "active" var, moves "engaged" var define to the parent of the station/hub.

Test firing a teleporter will now remove the 5% chance of spacing you for 3 seconds. (a bunch of people believed that this was already true, myself included)

Tested extensively.

### CODE REVIEW

**Comment by Konater** [comment]
Can you remove the chance entirely please?

**Comment by Intigracy** [comment]
Test fire it nerd.

**Comment by Shadowmech88** [comment]
I've never heard the phrase "test fire the teleporter."
I'm not sure exactly what you're talking about.

**Comment by Intigracy** [comment]
testfire

Currently it's useless.

The only thing it does is spawn sparks and use power.

File: code/game/objects/items/weapons/teleportation.dm [hunk 1]
```diff
@@ -151,7 +151,7 @@ Frequency:
 	var/list/L = list(  )
 	for(var/obj/machinery/computer/teleporter/R in machines)
 		for(var/obj/machinery/teleport/hub/com in locate(R.x + 2, R.y, R.z))
-			if(R.locked && !R.one_time_use)
+			if(R.locked)
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/teleportation.dm]: Why did you remove this?

Comment (Intigracy) [inline code/game/objects/items/weapons/teleportation.dm]: One time use is the thing that's a fragment from the clown planet card that
sent you to the clown planet on centcomm.

Comment (ComicIronic) [inline code/game/objects/items/weapons/teleportation.dm]: But why did you remove it?

Comment (Intigracy) [inline code/game/objects/items/weapons/teleportation.dm]: Because it's unused since clown planet got removed

Comment (ComicIronic) [inline code/game/objects/items/weapons/teleportation.dm]: Which doesn't mean nobody will ever use it - if it has a use case, you should leave it in.

**Comment by Intigracy** [comment]
Mmk.

**Comment by Intigracy** [comment]
Are there any outstanding code issues on this?

**Comment by Intigracy** [comment]
updated wiki
